### DISCUSSION TITLE
Web2print - WkHtmlToPdf unable to generate PDF

### DISCUSSION THREAD

**Discussion by philipproinerheraeus** [pimcore/pimcore#7871] [Q&A / Support] [answered]
Hi everyone!

I am not able to get the PDF-generation running.
The preview-tab shows everything perfectly right, but when I want to generate the PDF, the process remains at 0%.

I assume that it has to do something with the wkhtmltopdf Binary.
In the web2print-output.log I receive the message: "The system cannot find the specified path."

I am using Pimcore locally with XAMPP in Windows 10.
I installed wkhtmltopdf and it works within the command line.
For the wkhtmltopdf Binary setting I tried different approaches (both absolute and relative).

Does anybody has an idea what else I could do or what other path could be meant in the log?

Kind regards,
Philipp

**Comment by pdchaudhary** [comment, marked as answer]
@philipproinerheraeus 
Did you find any solution?

**Reply by neha04** [reply]
Facing same issue.  @dvesh3 its also not working on demo https://demo.pimcore.fun/admin

**Reply by pdchaudhary** [reply]
@neha04 Pdf Generation process is moved to queue so I think for that you need to enable cron process.

**Reply by pdchaudhary** [reply]
Check this: https://github.com/pimcore/demo/issues/277
